Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -498,7 +493,7 @@ private void GetChainedElectrocutionTargets(EntityUid source, List<(EntityUid en
 
     private void PlayElectrocutionSound(EntityUid targetUid, EntityUid sourceUid, ElectrifiedComponent? electrified = null)
     {
-        if (!Resolve(sourceUid, ref electrified) || !electrified.PlaySoundOnShock)
+        if (!Resolve(sourceUid, ref electrified, false) || !electrified.PlaySoundOnShock)
```

Comment:
Named boolean args please.

Reply:
For miscellaneous public methods, I agree that named bools should be used. However for a method as ubiquitous as `Resolve()` (and with only a single optional parameter), I think it just makes code more annoying to write & read.

Reply:
If it's >1 bool arg I would argue in favour but yeah for this would be better to lint or something if we really wanted (and like you said everyone knows resolve).